metadata_layer.py

```diff
@@ -513,37 +904,38 @@ def __init__(
             self.identifiers.append(self.primary_identifier)
         self.subjects = subjects or []
         self.contributors = contributors or []
-        self.links = links or []
         self.measurements = measurements or []
-        self.formats = formats or []
-        self.rights_uri = rights_uri
-        self.circulation = circulation
 
-        self.last_update_time = last_update_time
-        for link in self.links:
-            # If a link has a rights_uri, make that the overall rights_uri.
-            # If there are multiple links with a rights_uri, they should be
-            # split into separate metadata objects.
-            if link.rights_uri:
-                self.rights_uri = link.rights_uri
-
-            # An open-access link or open-access rights implies a FormatData object.
-            open_access_link = (link.rel == Hyperlink.OPEN_ACCESS_DOWNLOAD
-                                and link.href)
-            open_access_rights_link = (link.media_type in Representation.BOOK_MEDIA_TYPES
-                                       and link.href
-                                       and self.rights_uri in RightsStatus.OPEN_ACCESS)
-
-            if open_access_link or open_access_rights_link:
-                self.formats.append(
-                    FormatData(
-                        content_type=link.media_type,
-                        drm_scheme=DeliveryMechanism.NO_DRM,
-                        link=link
-                    )
-            )
+        # circulation-metadata connection has been removed, but might come back
+        #self.circulation = circulation
```

Comment:
it did.  which makes me wonder why the unit tests passed with that line commented out in the init.